tools/AWSFeedStorage.py

```diff
@@ -17,22 +19,63 @@ def __init__(self, uri):
 
         self.logger = logging.getLogger(__name__)
 
+        u = urlparse(uri)
+        self.s3_file = u.path[1:]
```

Comment:
`u.path[1:]` that seems like a hack. why are we getting all elements apart from 0.

Reply:
0 is just the aws:// to make scrapy point to this particular FeedStorage. The real file path starts at 1

Reply:
interesting. ok 👍